hs_core/models.py

```diff
@@ -229,7 +229,7 @@ class Meta:
 
 class Party(AbstractMetaDataElement):
     description = models.URLField(null=True, blank=True, validators=[validate_user_url])
-    name = models.CharField(max_length=100)
+    name = models.CharField(max_length=100, null=True, blank=True)
```

Comment:
Since ```name``` is optional now, the get_xml() function in hs_core/models.py also need to treat this attribute as optional.